gradle/libs.versions.toml

```diff
@@ -0,0 +1,92 @@
+[versions]
+androidGradlePlugin = "7.4.0"
+androidxActivity = "1.6.1"
+androidxAppCompat = "1.6.0"
+androidxComposeBom = "2023.01.00"
+androidxComposeCompiler = "1.4.0"
+androidxCore = "1.9.0"
+androidxCoreSplashscreen = "1.0.0"
+androidxDataStore = "1.0.0"
+androidxEspresso = "3.5.1"
+androidxLifecycle = "2.6.0-alpha04"
+androidxNavigation = "2.5.3"
+androidxTestCore = "1.5.0"
+androidxTestExt = "1.1.5"
+androidxTestRules = "1.5.0"
+androidxTestRunner = "1.5.1"
+coil = "2.2.2"
+firebaseAnalytics = "21.2.0"
+firebaseCrashlytics = "18.3.3"
+firebaseGradle = "2.9.2"
+googleServices = "4.3.15"
+junit4 = "4.13.2"
+koin = "3.3.2"
+koinCompose = "3.4.1"
+kotlin = "1.8.0"
+kotlinxCoroutines = "1.6.4"
+ksp = "1.8.0-1.0.9"
+moshi = "1.14.0"
+okhttp = "4.10.0"
+retrofit = "2.9.0"
+room = "2.5.0"
+timber = "5.0.1"
+
+[libraries]
+androidx-activity-compose = { group = "androidx.activity", name = "activity-compose", version.ref = "androidxActivity" }
```

Comment:
Having the names in `snake-case` allows us to access them by `.`. Example: `libs.androidx.activity.compose`
This also groups libraries under the same package. For example, anything related to `androidx` will be found under `libs.androidx`